keyboards/cybergear/macro25/info.json

```
@@ -0,0 +1,25 @@
+{
+    "keyboard_name": "Macro225 (mk-macro-01)",
+    "url": "https://github.com/CyberGear/mk-macro-01",
+    "maintainer": "CyberGear",
+    "usb": {
+        "vid": "0x1209",
+        "pid": "0x69A1",
+        "device_ver": "0x0001"
+    },
+    "width": 10,
+    "height": 2,
+    "diode_direction": "COL2ROW",
+    "layouts": {
+        "LAYOUT": {
```

I have a preference for these macro names being specific.

```suggestion
        "LAYOUT_ortho_2x5": {
```